[release] v6.0.0-alpha.3 (#41916)
mui/material-ui: CHANGELOG.md

# [Versions](https://mui.com/versions/)

## v6.0.0-alpha.3

<!-- generated comparing v6.0.0-alpha.2..next -->

_Apr 15, 2024_

A big thanks to the 19 contributors who made this release possible.

> This release was mostly about supporting CSS extraction and API standardization.

Reviewer: ```suggestion
This release was mostly about converting components to Pigment CSS and API standardization.
```

I'd use the "Pigment CSS" brand name as much as possible (to really strengthen it) instead of what it does (CSS extraction, which won't make folks immediately think of Pigment).

Author: Good idea ⭐

> This release was mostly about converting components to Pigment CSS and API standardization.

Author: ```suggestion
This release was mostly about converting components to Pigment CSS and API standardization.
```

> ### `@mui/material@6.0.0-alpha.2`

Reviewer 2: ```suggestion
### `@mui/material@6.0.0-alpha.3`
```

- [Button] Add `onChange` event handler to file upload example (#41863) @aarongarciah
- [Dialog] Prevent onClick on the root element from being overwritten (#41881) @ryanburr
- [FloatingActionButton] Convert to support CSS extraction (#41851) @gijsbotje
- Fix import statement in migration guide (#41852) @sai6855
- Convert `CircularProgress` to support Pigment CSS (#41776) @siriwatknp
- [PaginationItem] Convert to support CSS extraction (#41848) @gijsbotje
- [StepConnector] deprecate composed classes (#41740) @sai6855
- [StepLabel] Deprecate `StepIconComponent`, `StepIconProps` (#41835) @sai6855
- [ToggleButton] Convert to support CSS extraction (#41782) @lhilgert9
- [ToggleButtonGroup] Deprecate composed classes (#41288) @sai6855
- [Typography] Fix Typography inherit variant styles (#41308) @kealjones-wk

> 

Reviewer 2: ```suggestion
### `@mui/system@6.0.0-alpha.3`

- Add container queries utility (#41674) @siriwatknp
```

# [Versions](https://mui.com/versions/)

## v6.0.0-alpha.3

<!-- generated comparing v6.0.0-alpha.2..next -->

_Apr 17, 2024_

A big thanks to the 24 contributors who made this release possible.
This release was mostly about converting components to Pigment CSS and API standardization.

### `@mui/material@6.0.0-alpha.3`

- Convert `LinearProgress` to support Pigment CSS (#41816) @siriwatknp

> - [Button] Add `onChange` event handler to file upload example (#41863) @aarongarciah

Reviewer 3: This should be in docs section.

- [Dialog] Prevent onClick on the root element from being overwritten (#41881) @ryanburr
- [FloatingActionButton] Convert to support CSS extraction (#41851) @gijsbotje

> - Fix import statement in migration guide (#41852) @sai6855

Reviewer 3: This should be in docs section.

- Convert `CircularProgress` to support Pigment CSS (#41776) @siriwatknp
- [PaginationItem] Convert to support CSS extraction (#41848) @gijsbotje
- [StepConnector] deprecate composed classes (#41740) @sai6855
- [StepLabel] Deprecate `StepIconComponent`, `StepIconProps` (#41835) @sai6855
- [ToggleButton] Convert to support CSS extraction (#41782) @lhilgert9
- [ToggleButtonGroup] Deprecate composed classes (#41288) @sai6855
- [Typography] Fix Typography inherit variant styles (#41308) @kealjones-wk

### `@mui/system@6.0.0-alpha.3`

> - Add container queries utility (#41674) @siriwatknp

Reviewer 3: Could this be a highlight?

Author: What do you think @siriwatknp?

Reviewer 2: Sure, it can be a highlight.
